input/data/2008/2008ApJ...679.1427A/tev-000014-sed.ecsv

```diff
@@ -0,0 +1,24 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
```

Comment:
I see that the paper doesn't give a table with the spectral point information (like most papers). So it's not like in this case `e_min` or `e_max` or `e_ref` or `flux` or `dnde` or anything was really "published". The spectral points shown in Figure 2 are `dnde` drawn at some `e_ref` values.

Irrespective of the question if you add the `e_ref` column or not (we'll merge either way): can you add a "comment" or "url" in the header where this data is coming from?
If you look at other examples you'll see that it's always either a "url" pointing to a webpage or file with the information, or a "comment" saying something about the origin of the data (in this case I presume a VERITAS-internal webpage or from corresponding author).